packages/interactivity-router/src/head.ts

```diff
@@ -29,6 +34,14 @@ export const updateHead = async ( newHead: HTMLHeadElement[] ) => {
 		}
 	}
 
+	await Promise.all(
+		[ ...headElements.entries() ]
+			.filter( ( [ , { tag } ] ) => tag.nodeName === 'SCRIPT' )
+			.map( async ( [ url ] ) => {
+				await import( /* webpackIgnore: true */ url );
```

Comment:
We need the webpack comment here. Otherwise, webpack will try to code-split here into a new chunk and it [does not permit](https://webpack.js.org/api/module-methods/#dynamic-expressions-in-import) using fully dynamic paths.